packages/tools/generate-images.ts

```diff
@@ -33,6 +33,22 @@ interface Results {
 	done: Record<string, boolean>;
 }
 
+interface AppIconContents {
+	images: {
+		appearances?: [
+			{
+				appearance: 'luminosity';
+				value: 'dark' | 'tinted';
+			},
+		];
+		filename?: string;
+		idiom: 'universal';
+		platform: 'ios';
+		scale?: '2x' | '3x';
+		size: string;
+	}[];
```

Comment:
Took me a while to make sense of these new AppIcon things because it looks generic but it's really just for iOS, right? In that case please scope these interfaces and constants by prefixing them with "iOS"

Reply:
Yes, its the `Contents.json` file for the ios AppIcon set.